devel/README.md

```diff
@@ -255,6 +263,38 @@ tooling functions correctly
 (You can safely ignore some errors like
 `com_github_opencontainers_go_digest Does not exist`)
 
+### Building the containers
+
+This tool builds and publishes Mixer container images to the specified
+registry.
+
+```
+bin/publish-docker-images.sh -h gcr.io/my-project -t my-tag
+```
+
+where
+
+* The `-h` parameter `my-project` is the composition of the registry hostname
+  and the project id. This should be customized.
+* The `-t` parameter `my-tag` is the desired tag. This should be customized.
+
+### Building the Istio manfiests
+
+Use [updateVersion.sh](https://github.com/istio/istio/blob/master/install/updateVersion.sh)
+to generate new manifests with the specified Mixer containers.
+
+```
+cd $ISTIO/istio
+install/updateVersion.sh -r gcr.io/my-project,tag
```

Comment:
one of these doesn't work well if you put a space (https://github.com/istio/istio/issues/642) are you sure this works as is ?

Reply:
very odd; thanks for the catch!  i'll update the PR now.